### PR TITLE
chore: github form templates

### PR DESCRIPTION
**Pull request by crazy-max**
similar to https://github.com/docker/buildx/pull/1037
~needs #931~

image

image

image

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      label: Contributing guidelines
+      description: >
+        Please read the contributing guidelines before proceeding.
+      options:
+        - label: I've read the [contributing guidelines](https://github.com/docker/build-push-action/blob/master/.github/CONTRIBUTING.md) and wholeheartedly agree
+          required: true
+
+  - type: checkboxes
+    attributes:
+      label: I've found a bug and checked that ...
```
Comment (dvdksn) [inline .github/ISSUE_TEMPLATE/bug.yml:26]:
```suggestion
      label: "I've found a bug, and:"
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      description: |
+        Make sure that your request fulfills all of the following requirements.
+        If one requirement cannot be satisfied, explain in detail why.
+      options:
+        - label: ... the documentation does not mention anything about my problem
+        - label: ... there are no open or closed issues that are related to my problem
```
Comment (dvdksn) [inline .github/ISSUE_TEMPLATE/bug.yml:32]:
```suggestion
        - label: The documentation does not mention anything about my problem
        - label: There are no open or closed issues that are related to my problem
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+  - type: textarea
+    attributes:
+      label: Description
+      description: >
+        Please provide a brief description of the bug in 1-2 sentences.
```
Comment (dvdksn) [inline .github/ISSUE_TEMPLATE/bug.yml:38]:
```suggestion
        Provide a brief description of the bug in 1-2 sentences.
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Expected behaviour
+      description: >
+        Please describe precisely what you'd expect to happen.
```
Comment (dvdksn) [inline .github/ISSUE_TEMPLATE/bug.yml:46]:
```suggestion
        Describe precisely what you'd expect to happen.
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Actual behaviour
+      description: >
+        Please describe precisely what is actually happening.
```
Comment (dvdksn) [inline .github/ISSUE_TEMPLATE/bug.yml:54]:
```suggestion
        Describe precisely what is actually happening.
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+  - type: input
+    attributes:
+      label: Repository URL
+      description: >
+        Please provide the URL of the repository where you are experiencing the
+        issue. If your repository is private please provide a link to a minimal
+        repository that reproduces the issue. If you are not able to share any
+        of the above please provide your YAML workflow (asked below) and
+        make sure to remove any sensitive information.
```
Comment (dvdksn) [inline .github/ISSUE_TEMPLATE/bug.yml:66]:
```suggestion
        Enter the URL of the repository where you are experiencing the
        issue. If your repository is private, provide a link to a minimal
        repository that reproduces the issue.
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+
+  - type: input
+    attributes:
+      label: Workflow run URL
+      description: >
+        Same as above but for the workflow run URL. If you are not able to
+        share any of the above please provide at least the logs of the workflow
+        run URL (asked below) and make sure to remove any sensitive information.
```
Comment (dvdksn) [inline .github/ISSUE_TEMPLATE/bug.yml:74]:
```suggestion
        Enter the URL of the GitHub Action workflow run, if public.
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+
+  - type: textarea
+    attributes:
+      label: YAML workflow
+      description: >
+        Please provide the YAML workflow and make sure to remove any sensitive
+        information (if applicable).
```
Comment (dvdksn) [inline .github/ISSUE_TEMPLATE/bug.yml:81]:
```suggestion
        Provide the YAML of the workflow that's causing the issue.
        Make sure to remove any sensitive information.
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+      render: yaml
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Workflow logs
+      description: >
+        Please [attach](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/attaching-files)
```
Comment (dvdksn) [inline .github/ISSUE_TEMPLATE/bug.yml:90]:
```suggestion
        [Attach](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/attaching-files)
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 10]
```diff
@@ -0,0 +1,105 @@
...
+        the [log file of your workflow run](https://docs.github.com/en/actions/managing-workflow-runs/using-workflow-run-logs#downloading-logs)
+        and make sure to remove any sensitive information (if applicable).
```
Comment (dvdksn) [inline .github/ISSUE_TEMPLATE/bug.yml:92]:
```suggestion
        and make sure to remove any sensitive information.
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 11]
```diff
@@ -0,0 +1,105 @@
...
+
+  - type: textarea
+    attributes:
+      label: BuildKit logs
+      description: >
+        If applicable, please provide the [BuildKit container logs](https://docs.docker.com/build/ci/github-actions/configure-builder/#buildkit-container-logs)
```
Comment (dvdksn) [inline .github/ISSUE_TEMPLATE/bug.yml:98]:
```suggestion
        If applicable, provide the [BuildKit container logs](https://docs.docker.com/build/ci/github-actions/configure-builder/#buildkit-container-logs)
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 12]
```diff
@@ -0,0 +1,105 @@
...
+      value: |
+        Thank you for taking the time to report a bug!
+        If this is a security issue please report it to the [Docker Security team](mailto:security@docker.com).
+        Before submitting a bug report please read the [Troubleshooting doc](https://github.com/docker/build-push-action/blob/master/TROUBLESHOOTING.md).
+
+  - type: checkboxes
+    attributes:
+      label: Contributing guidelines
+      description: >
+        Please read the contributing guidelines before proceeding.
```
Comment (dvdksn) [inline .github/ISSUE_TEMPLATE/bug.yml:19]:
```suggestion
        Make sure you've read the contributing guidelines before proceeding.
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 13]
```diff
@@ -0,0 +1,105 @@
...
+labels:
+  - status/triage
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to report a bug!
+        If this is a security issue please report it to the [Docker Security team](mailto:security@docker.com).
+        Before submitting a bug report please read the [Troubleshooting doc](https://github.com/docker/build-push-action/blob/master/TROUBLESHOOTING.md).
```
Comment (dvdksn) [inline .github/ISSUE_TEMPLATE/bug.yml:13]:
```suggestion
        Before submitting a bug report, check out the [Troubleshooting doc](https://github.com/docker/build-push-action/blob/master/TROUBLESHOOTING.md).
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 14]
```diff
@@ -0,0 +1,105 @@
...
+      label: BuildKit logs
+      description: >
+        If applicable, please provide the [BuildKit container logs](https://docs.docker.com/build/ci/github-actions/configure-builder/#buildkit-container-logs)
+      render: text
+
+  - type: textarea
+    attributes:
+      label: Additional info
+      description: |
+        Please provide any additional information that could be useful.
```
Comment (dvdksn) [inline .github/ISSUE_TEMPLATE/bug.yml:105]:
```suggestion
        Provide any additional information that could be useful.
```

**Review by dvdksn — COMMENTED** [review]
LGTM, just some non-blocking suggestions

**Review by dvdksn — APPROVED** [review]

**Review by jedevc — APPROVED** [review]

**Pull request merged**